lib/dotcom_web/components/trip_planner/itinerary_detail.ex

```diff
@@ -20,6 +20,28 @@ defmodule DotcomWeb.Components.TripPlanner.ItineraryDetail do
     assigns =
       assign(assigns, :selected_itinerary, Enum.at(itineraries, selected_itinerary_detail_index))
 
+    ~H"""
+    <div>
+      <.depart_at_buttons
+        selected_itinerary_detail_index={@selected_itinerary_detail_index}
+        itineraries={@itineraries}
+        target={@target}
+      />
+      <.specific_itinerary_detail itinerary={@selected_itinerary} />
+    </div>
+    """
+  end
+
+  attr :itineraries, :list
+  attr :selected_itinerary_detail_index, :integer
+  attr :target, :string
+
+  defp depart_at_buttons(%{itineraries: [_]} = assigns) do
+    ~H"""
+    """
+  end
+
+  defp depart_at_buttons(assigns) do
     ~H"""
     <div>
```

Comment:
```suggestion
    <div :if={Enum.count(@itineraries) < 2}>
```
😅 I feel like adding the `if` directive to the existing markup would suffice (and be a bit more readable)? Unless I'm missing something?

Reply:
Nope! I just got into pattern-matching brain. The `:if` directive is easier to read in this context!

https://github.com/mbta/dotcom/pull/2250/commits/09081ebf157e86d49d93b7a5d8de077d379e24ef